src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/AssemblyAttributeProperties/SourceAssemblyAttributePropertyValueProvider.cs

```diff
@@ -110,16 +114,24 @@ public async Task SetPropertyValueAsync(string value)
         /// </summary>
         private static async Task<AttributeData?> GetAttributeAsync(string assemblyAttributeFullName, Project project)
         {
-            Compilation compilation = await project.GetCompilationAsync();
-            ImmutableArray<AttributeData> assemblyAttributes = compilation.Assembly.GetAttributes();
+            if (project == null)
+            {
+                return null;
+            }
+            Compilation? compilation = await project.GetCompilationAsync();
+            ImmutableArray<AttributeData>? assemblyAttributes = compilation?.Assembly.GetAttributes();
 
-            INamedTypeSymbol attributeTypeSymbol = compilation.GetTypeByMetadataName(assemblyAttributeFullName);
+            INamedTypeSymbol? attributeTypeSymbol = compilation?.GetTypeByMetadataName(assemblyAttributeFullName);
             if (attributeTypeSymbol == null)
             {
                 return null;
             }
 
-            return assemblyAttributes.FirstOrDefault((data, symbol) => data.AttributeClass.Equals(symbol), attributeTypeSymbol);
+#pragma warning disable CS8602 // Dereference of a possibly null reference.
```

Comment:
Why the suppressions? Are these false positives?

Reply:
detected as a warning.